package.json

```diff
@@ -22,8 +22,12 @@
 		"lint:eslint:scripts": "eslint --cache --fix scripts",
 		"lint:eslint:tests": "eslint --cache --fix tests",
 		"lint:eslint": "npm-run-all --continue-on-error --parallel lint:eslint:src",
+		"test": "echo 'No Tests yet'",
 		"prepare": "husky install",
-		"test": "echo 'No Tests yet'"
+		"pub:patch": "npm version patch && rm -rf dist && yarn pub",
```

Comment:
Does `patch` automatically bump the "version" in the package.json from say 1.0.0 to 1.0.1, and publish it to artifcatory with that new version?

Reply:
Yup